05-formatting.Rmd

```diff
@@ -205,32 +205,79 @@ The chunk option `fig.align`\index{chunk option!fig.align} specifies the alignme
 
 Typically we write code chunks and inline expressions that we want to be parsed and evaluated by **knitr**. However, if you are trying to write a tutorial on using **knitr**, you may need to generate a verbatim code chunk or inline expression that is _not_ parsed by **knitr**, and we want to display the content of the chunk header. 
 
-Unfortunately, we cannot wrap the code chunk in another layer of backticks, but instead we must make the code chunk invalid within the source code by inserting `` `r knitr::inline_expr("''")` ``\index{knitr!inline\_expr()} in the chunk header. This will be evaluated as an inline expression to _an empty string_ by **knitr**. For this example, the following "code chunk" in the source document:
+The `verbatim` engine will take any type of R Markdown valid content and output it as-is in the resulting document. By adding one more backtick in the upper layer than the inside content, a whole code chunk including the chunk header can be included. 
 
-````{r echo = FALSE, comment = NA}
-cat("```{r, eval=TRUE}`r ''`
+`````{verbatim}
+````{verbatim}
+```{r, eval=TRUE}
 1 + 1
-```")
+```
 ````
+`````
 
-will be rendered as:
+will be rendered as: 
 
+````{verbatim}
+```{r, eval=TRUE}
+1 + 1
+```
 ````
-```{r, eval=TRUE}`r ''`
+
+in the output. You have to use at least N+1 backticks to wrap up N backticks, so four backticks here for the chunk header with the `verbatim` engine. 
```

Comment:
```suggestion
in the output. You have to use at least `N+1` backticks to wrap up `N` backticks. We are using four backticks here for the `verbatim` chunk because the inner chunk header contains three backticks.
```